<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMediaSourceDlg</class>
 <widget class="QDialog" name="AddMediaSourceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AddMediaSourceDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wgt_main{
background:#262626;
border:1px solid #585858;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wgt_main" native="true">
     <property name="styleSheet">
      <string notr="true">#wgt_top{
background:#2F2F2F;
}
#wgt_middle{
background:#262626;
}
#wgt_bottom{
background:#262626;
border:1px solid #2F2F2F;
border-left:none;
border-right:none;
border-bottom:none;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_top" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#button_close{
background:transparent;
background-image: url(:/TQLive/images/pb_close_pressed.png);
background-repeat:no-repeat;
border:none;
}
#button_close:hover{
background-image: url(:/TQLive/images/pb_close_normal.png);
}
#button_close:pressed{
margin:1px;
padding:1px;
}
QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#4AB134;
background:transparent;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>19</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_title">
           <property name="text">
            <string>添加视频</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_close">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="wgt_middle">
        <property name="styleSheet">
         <string notr="true">#page_addMedia,#page_mediaList{
background:#262626;
}
#listWidget_playList{
  background:transparent;
  outline:0px;
}
#listWidget_playList::item:selected{
background:#2F2F2F;
}
#listWidget_playList::item:hover{
background:#2F2F2F;
}
</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_addMedia">
         <property name="styleSheet">
          <string notr="true">#label_tips{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;
background:transparent;
}
#button_addMediaFirst{
background:transparent;
border-image: url(:/TQLive/images/add_media_default.png);
}
#button_addMediaFirst:hover{
border-image: url(:/TQLive/images/add_media_hover.png);
}
#button_addMediaFirst:pressed{
margin:1px;
padding:1px;
}</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>150</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>176</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>14</number>
            </property>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="button_addMediaFirst">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_tips">
              <property name="text">
               <string>系统自动开启列表循环播放功能</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>176</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>150</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_mediaList">
         <property name="styleSheet">
          <string notr="true">QLabel,QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
background:transparent;
}
#label_playList{
color:#999999;
}
#button_addMedia{
color:#4AB134;
padding:0px;
}
#button_addMedia:hover{
color:#999999;
}
#listWidget_playList{
background:#262626;
border:none;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>40</number>
          </property>
          <property name="topMargin">
           <number>25</number>
          </property>
          <property name="rightMargin">
           <number>40</number>
          </property>
          <property name="bottomMargin">
           <number>25</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_playList">
              <property name="text">
               <string>播放列表</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_addMedia">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>26</height>
               </size>
              </property>
              <property name="text">
               <string>+ 添加视频</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_playList">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgt_bottom" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel,QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#999999;
background:transparent;
}
#button_last{
border-image: url(:/TQLive/images/last_default.png);
}
#button_last:hover{
border-image: url(:/TQLive/images/last_hover.png);
}
#button_last:disable{
border-image: url(:/TQLive/images/last_disabled.png);
}
#button_last:pressed{
margin:1px;
padding:1px;
}
#button_next{
border-image: url(:/TQLive/images/next_default.png);
}
#button_next:hover{
border-image: url(:/TQLive/images/next_hover.png);
}
#button_next:disable{
border-image: url(:/TQLive/images/next_disabled.png);
}
#button_next:pressed{
margin:1px;
padding:1px;
}
#button_sound{
border-image: url(:/TQLive/images/live tool_icon_the horn_normal.png);
}
#button_sound:hover{
border-image: url(:/TQLive/images/live tool_icon_the horn_hover.png);
}
#button_sound:checked{
border-image: url(:/TQLive/images/live tool_icon_mute_normal.png);
}
#button_sound:checked:hover{
border-image: url(:/TQLive/images/live tool_icon_mute_hover.png);
}
QSlider{
background:transparent;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>29</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>29</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_totalTime">
             <property name="text">
              <string>00:00:00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slider_play">
             <property name="minimumSize">
              <size>
               <width>390</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>390</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_currTime">
             <property name="text">
              <string>00:00:00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>11</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>20</number>
             </property>
             <item>
              <widget class="QPushButton" name="button_last">
               <property name="minimumSize">
                <size>
                 <width>9</width>
                 <height>12</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>9</width>
                 <height>12</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_next">
               <property name="minimumSize">
                <size>
                 <width>9</width>
                 <height>12</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>9</width>
                 <height>12</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_sound">
             <property name="minimumSize">
              <size>
               <width>13</width>
               <height>11</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>13</width>
               <height>11</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slider_sound">
             <property name="minimumSize">
              <size>
               <width>88</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>88</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="CustomLoopButton" name="button_loop">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomLoopButton</class>
   <extends>QPushButton</extends>
   <header>CustomLoopButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
